draft-meynell-panrg-scion-research-questions.md

```diff
@@ -196,9 +196,34 @@ This draft assumes the reader is familiar with some of the fundamental concepts
 
 ## ISD, AS identity
 
+The SCION protocol specifies 16 bits and 48 bits to identify the ISD and AS.
+This identification is, at this moment, necessary at the data-plane level.
+It is written in each one of the data packets to fully address the sender and receiver,
+as well as at the control-plane level, to identify the PCB sender and hops.
+
+While 48 bits seem sufficient to accommodate a large number of ASes,
+using 16 bits for the ISD might not be.
+Additionally, the justification of these figures is lacking at the moment.
```

Comment:
Can this be better clarified? Is there any more information about why these values were chosen.

Reply:
Honestly I don't know why they were chosen as 16-48. In the book we also don't say.

Reply:
>  which is conceivably insufficient 

Wether this is sufficient or not depends on how these would be assigned.. as you say we don't have concrete justifications on these figures, so we also have no justifications wether this is sufficient or not. I'd therefore change language a bit from: 
> using 16 bits for the ISD only offers 65,536 possible assignments which is conceivably insufficient.
Additionally, the justification of these figures is lacking at the moment.

To
> using 16 bits for the ISD only offers 65,536 possible assignments. Further investigation on whether thisis sufficient is needed.